plugins/inputs/nfs/nfs.go

```diff
@@ -0,0 +1,344 @@
+package nfs
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NFS struct {
+	Iostat   bool
+	Fullstat bool
+}
+
+var sampleConfig = `
```

Comment:
Why does `iostat = true` mean that the plugin will read fewer metrics?? Can you clarify these two config options?